core/Tracker/Visit/VisitProperties.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tracker\Visit;
+
+/**
+ * Holds temporary data for tracking requests.
+ *
+ * RequestProcessors
+ */
+class VisitProperties
```

Comment:
I understand now, thx. Not sure if any plugin developer would understand it without knowing the background. I'm not sure if a plugin developer has to understand the difference between visit and visitor. Also to keep in mind but not really relevant: Soon we will not have `Visitor` and `Visit` anymore but `User` and `Session` or even something else (as also machines could be tracked etc). I'm not happy about `VisitProperties` as it doesn't sound intuitive to me and really struggled understanding it whenever I saw it but right now I don't come up with a better solution. I'd personally would probably prefer `Visitor` even though it's not 100% correct but easier to read but I also understand that someone doesn't want to use it. Maybe we can think about it again when it becomes a public API, by then we might have other terms anyway (eg User and Session)